services/visual_recognition/v3.js

```diff
@@ -350,6 +334,59 @@ module.exports = function (RED) {
     });
   }
 
+  // This function performs the operation to Delete ALL Dialogs
+  function performDeleteAllClassifiers(visualRecognition, node, msg) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['performDeleteAllClassifiers' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2713769514/issues/source?bid=3225115&fileBranchId=3344975#l338)